### PR TITLE
Use NS_OPTIONS for PINRemoteImageManagerDownloadOptions

### PR DESCRIPTION
**Pull request by Adlai-Holler**
Just starting to use this awesome framework, and really excited!

### CODE REVIEW

File: Pod/Classes/PINRemoteImageManager.h [hunk 1]
```diff
@@ -35,7 +35,7 @@ typedef NS_ENUM(NSUInteger, PINRemoteImageManagerError) {
 /**
  Options with which to download and process images
  */
-typedef NS_ENUM(NSUInteger, PINRemoteImageManagerDownloadOptions) {
+typedef NS_OPTIONS(NSUInteger, PINRemoteImageManagerDownloadOptions) {
     /** Download and process with default options (no other options set) */
     PINRemoteImageManagerDownloadOptionsNone = 0,
     /** Regardless of the image type downloaded, return UIImages and *not* FLAnimatedImage */
```
Comment (garrettmoon) [inline Pod/Classes/PINRemoteImageManager.h]: Should we get rid of all the bit mask stuff below if we switch to NS_OPTIONS?

**Comment by garrettmoon** [comment]
Thanks!

File: Pod/Classes/PINRemoteImageManager.h (hunk 1 above)
Comment (Adlai-Holler) [inline Pod/Classes/PINRemoteImageManager.h]: Yep, doing that. One minute.

**Comment by Adlai-Holler** [comment]
Happy to do it man!

**Comment by garrettmoon** [comment]
Hey @Adlai-Holler I actually gave this a test and I was wrong, we still need  the explicit options values, NS_OPTIONS *doesn't* do this for us. It appears that using NS_OPTIONS is functionally nearly the same as NS_ENUM but it notifies the compiler that it's a bit mask (for what reason, I don't know).

TL;DR; Before this can be merged, we need the original pull request you made back :) Please reinsert the values for each of the constants.

**Comment by Adlai-Holler** [comment]
@garrettmoon Huh, are you completely 100% sure? I just tested this code in AppDelegate:

```objectivec
    NSLog(@"%d", PINRemoteImageManagerDownloadOptionsIgnoreGIFs);
    NSLog(@"%d", PINRemoteImageManagerDownloadOptionsSkipDecode);
    NSLog(@"%d", PINRemoteImageManagerDownloadOptionsSkipEarlyCheck);
    NSLog(@"%d", PINRemoteImageManagerSaveProcessedImageAsJPEG);
```

And the output was 

```
2015-11-23 11:47:20.424 PINRemoteImage[57822:9734554] 1
2015-11-23 11:47:20.424 PINRemoteImage[57822:9734554] 2
2015-11-23 11:47:20.424 PINRemoteImage[57822:9734554] 4
2015-11-23 11:47:20.424 PINRemoteImage[57822:9734554] 8
```

**Comment by garrettmoon** [comment]
I'm also using 7.1.1:

2015-11-23 13:39:41.842 PINRemoteImage[47298:21324320] 1
2015-11-23 13:39:41.843 PINRemoteImage[47298:21324320] 2
2015-11-23 13:39:41.843 PINRemoteImage[47298:21324320] 3
2015-11-23 13:39:41.843 PINRemoteImage[47298:21324320] 4

Very strange!

**Comment by Adlai-Holler** [comment]
O_O That is crazy-pants. Ready to rock!
